PMM-9148 QAN share link (#700)
percona/pmm-doc: docs/using/query-analytics.md

Query Analytics can monitor MongoDB queries. Although MongoDB is not a relational database management system, you analyze its databases and collections in the same interface using the same tools.

> ## Sharing a link to Query Analytics

Reviewer: ```suggestion
## Sharing a link for Query Analytics
```

> To share a link to Query Analytics use the *Copy Link* button. This will copy a link to the clipboard with all relevant information so when the link is opened it shows exactly as you were seeing. This will copy information such as the selected query, table page, selected filters, details tab and time range.

Reviewer: ```suggestion
To share a link for Query Analytics, use *Copy Link*. It copies the link to the clipboard with all the relevant information such as selected query, table page, selected filters, details tab, and time range. Thus, when you open the link, it will display all the exact information.
```

!!! caution alert alert-warning ""

>     If you copy the link from the browser's address bar or use the standard functionality from Grafana to share a dashboard, Query Analytics might not open with the same information you were seeing at the moment of sharing.

Reviewer: ```suggestion
  Ensure that you use *Copy Link* to copy the link instead of using the browser address bar or the standard Grafana functionality (to share a dashboard). Otherwise, Query Analytics might not display the exact information that existed while sharing the link.
```

>     By default Grafana uses relative time range and not an absolute one, so when the copied link is opened it will have a different time.

Reviewer: ```suggestion
  By default, Grafana uses a relative time range and not an absolute range, so it will have a different timestamp when this link is opened.
```
